extensions/sql-migration/src/wizard/accountsSelectionPage.ts

```diff
@@ -49,7 +49,7 @@ export class AccountsSelectionPage extends MigrationWizardPage {
 			})
 			.withValidation((c) => {
 				if (c.value) {
-					if ((<azdata.CategoryValue>c.value).displayName === constants.ACCOUNT_SELECTION_PAGE_NO_LINKED_ACCOUNTS_ERROR) {
+					if ((<azdata.CategoryValue>c.value)?.displayName === constants.ACCOUNT_SELECTION_PAGE_NO_LINKED_ACCOUNTS_ERROR) {
```

Comment:
You're already checking that c.value exists (well..is truthy. But same thing in this case) in the if - this shouldn't be necessary.